helix-core/src/main/java/org/apache/helix/manager/zk/client/HelixZkClient.java

```diff
@@ -172,6 +186,15 @@ void asyncSetData(final String path, Object datat, final int version,
 
   long getSessionId();
 
+  /**
+   * Gets a session id in hexadecimal notation from a session id in long type.
+   * Ex. 0x1000a5ceb930004 is returned.
+   *
+   * @param sessionId session id in long type
+   * @return String representation of session id in hexadecimal notation.
+   */
+  String getHexSessionId(long sessionId);
```

Comment:
Either make it return directly, i.e, getHexSessionId(), otherwise, this is more like a util function (just doing a conversion, it may not  be  in ZkClient. or at least, if you think it should be util function somehow, then it should be defined something like:  static String toHexSessionId(long sessionId);